source/server/http/admin.cc

```diff
@@ -735,7 +746,10 @@ Http::Code AdminImpl::handlerStats(absl::string_view url, Http::HeaderMap& respo
   const Http::Utility::QueryParams params = Http::Utility::parseQueryString(url);
 
   const bool used_only = params.find("usedonly") != params.end();
-  const absl::optional<std::regex> regex = filterParam(params);
+  absl::optional<std::regex> regex;
+  if (!filterParam(params, response, regex)) {
+    return Http::Code::OK;
```

Comment:
Should we return a 400 (bad request)?

Reply:
I have not been doing that in the admin endpoints, because if you use 'wget' you don't get the response text that way.

Reply:
actually I guess there's a bunch of other places in admin where we return a 400; i'll switch it